bbb/services.py

```diff
@@ -457,6 +463,20 @@ def _isAuthorized(self, buildername, scopes):
         else:
             return True
 
+    def _refreshAllowedBuilders(self):
+        now = arrow.now().timestamp
+        if self.allowed_builders is None or (now - self.allowed_builders_age) > 300:
+            try:
+                builders = requests.get(ALL_THE_THINGS_URL).json()['builders']
```

Comment:
On the other hand the check (`x in y`) would fail, so we shouldn't be discarding messages.